src/main/java/com/microsoft/lst_bench/client/JDBCConnection.java

```diff
@@ -29,10 +30,12 @@ public class JDBCConnection implements Connection {
 
   private final java.sql.Connection connection;
   private final int maxNumRetries;
+  private final boolean verboseLogging;
 
-  public JDBCConnection(java.sql.Connection connection, int maxNumRetries) {
+  public JDBCConnection(java.sql.Connection connection, int maxNumRetries, boolean verboseLogging) {
```

Comment:
After reviewing the code, I'm wondering if it's more appropriate to rename this property to `showWarnings` (accordingly in the config file), since this is concretely what we are doing.